merlin/systems/dag/ops/feast.py

```diff
@@ -185,6 +186,13 @@ def __init__(
         self.store = FeatureStore(repo_path=repo_path)
         super().__init__()
 
+    def __getstate__(self):
+        # feature store objects aren't picklable - exclude from saved representation
```

Comment:
We don't want to try to pickle anything that either doesn't play nicely with pickle or that we could just as easily save to disk, so we exclude external files that can be loaded from disk by the ops.